index.html

```diff
@@ -7,6 +7,7 @@
 
 <div class="container target">
 	<h1>Welcome to the openHAB 2 Documentation!</h1>
+	<p>The <strong>open H</strong>ome <strong>A</strong>utomation <strong>B</strong>us (openHAB, <em>pronounced ˈəʊpənˈhæb</em>) is an open source, technology agnostic home automation platform which runs as the center of your smarthome!</p>
 	<p>This documentation contains different sections. If you are new to openHAB 2, you might want to start with one of our <a href="tutorials">Tutorials</a> that guides you through different steps by an example. If you are looking for some specific information on how to use openHAB, the <a href="overview.html">User Manual</a> is the right place to go and if you plan to get started with your own developments, please refer to our <a href="developers">Developer Guide</a>.</p>
```

Comment:
While you are at it. The first sentence seems like an awkward mixture of plural and singular